docs/how-to/Programmers_Guide.rst

```diff
@@ -900,6 +900,12 @@ There are three client executables that can be used with rocBLAS. They are:
 
 These three clients can be built by following the instructions in the Building and Installing section of the User Guide. After building the rocBLAS clients, they can be found in the directory ``rocBLAS/build/release/clients/staging``.
 
+.. note::
+  The ``rocblas-bench`` and ``rocblas-test`` executables use AMD's ILP64 version of AOCL-BLAS 4.2 as the host reference BLAS to verify correctness. However, there is a known issue with AOCL-BLAS that can cause these executables to hang. This problem can arise because the AOCL-BLAS library launches multiple threads to perform computations. If the number of threads matches the total number of CPU threads, it can lead to thread oversubscription, causing the program to hang.
```

Comment:
Sorry missed this earlier "total number of CPU threads" -> "total number of CPU logical cores"